### PR TITLE
Issue #2903008 by writes: Allow queue to process continuous job

### PR DESCRIPTION
**Pull request by writes**
https://www.drupal.org/node/2903008

Add the ability to process jobs that take multiple runs to complete.

### CODE REVIEW

**Review by WidgetsBurritos — APPROVED** [review]
👍 LGTM

File: src/Plugin/QueueWorker/CaptureQueueWorker.php [hunk 1]
```diff
@@ -18,26 +18,21 @@ class CaptureQueueWorker extends QueueWorkerBase {
...
+      if (empty($data[$key])) {
+        throw new \Exception("$key is required");
       }
+    }
 
-      // Capture the response.
-      $data['capture_response'] = $data['utility']->capture($data)->getResponse();
+    // Capture the response.
+    $data['capture_response'] = $data['utility']->capture($data)->getResponse();
+    if(isset($data['capture_response'])){
```
Comment (pobtastic) [inline src/Plugin/QueueWorker/CaptureQueueWorker.php]: Should be a space after the control statement here, and one before the curly brace.

**Comment by pobtastic** [comment]
I'm a +1, although there's one line which is missing some whitespace for Drupal coding standards.

**Review by WidgetsBurritos — APPROVED** [review]
👍 on 652fe85f5ca64bf6dd920252eeaa74257f2594ea

**Pull request merged**